lib/heidrun/id_source.rb

```diff
@@ -0,0 +1,55 @@
+
+module Heidrun
+
+  ##
+  # A source of identifier strings to use in constructing API requests.
```

Comment:
What are the expectations of the format of this file? This should be documented here.